tests/test_util.go

```diff
@@ -231,9 +236,30 @@ func waitUntilVersionUp(cli driver.Client, allowNoLeaderResponse ...bool) error
 		return maskAny(noLeaderErr)
 	}
 
+	if predicate != nil {
+		return predicate(vInfo)
+	}
+
 	return nil
 }
 
+// creates predicate to be used in waitUntilVersionUp
+func createEqualVersionsPredicate(info driver.VersionInfo) func(driver.VersionInfo) error {
```

Comment:
I would expect this function to have a `version driver.Version` argument, since that is the only thing you're comparing.
Will automatically make `createEqualVersionsPredicateFromString` obsolete.